src/frontends/onnx/tests/models/multinomial.prototxt

```diff
@@ -0,0 +1,49 @@
+ir_version: 7
+graph {
+  node {
+    input: "input"
+    output: "output"
+    op_type: "Multinomial"
+    attribute {
+      name: "sample_size"
+      i: 1
+      type: INT
+    }
+  }
+  name: "test_multinomial"
+  input {
+    name: "input"
+    type {
+      tensor_type {
+        elem_type: 1
+        shape {
+          dim {
+            dim_value: -1
+          }
+          dim {
+            dim_value: 10
+          }
+        }
+      }
+    }
+  }
+  output {
+    name: "output"
+    type {
+      tensor_type {
+        elem_type: 7
+        shape {
+          dim {
+            dim_value: -1
+          }
+          dim {
+            dim_value: 1
+          }
+        }
+      }
+    }
+  }
+}
+opset_import {
+  version: 1
```

Comment:
Multinomial introduce in opset-7. Right now it will work only due to known issue in FrontEnd :(